app/package.json

```diff
@@ -42,6 +42,7 @@
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
         "react-mentions": "^4.1.1",
+        "react-popup": "^0.10.0",
```

Comment:
If you didn't end up using this then remove it from the dependencies.

Reply:
Have done